src/memex/search/index.py

```diff
@@ -141,13 +143,16 @@ def index(self, annotation_ids=None):
         errored = set()
         for ok, item in indexing:
             if not ok:
-                errored.add(item['index']['_id'])
+                status = item[self.op_type]
+                if self.op_type != 'create' and \
```

Comment:
I'm not sure this conditional is right...

```python
was_index_op = self.op_type != 'create'
was_other_error = 'document already exists' not in status['error']

if was_index_op and was_other_error:
    record(error)
```

Doesn't that mean that *all* errors thrown on 'create' will be swallowed?

I'd think that something like this would be clearer:

```
was_doc_exists_err = 'document already exists' in status['error']
if self.op_type == 'create' and was_doc_exists_err:
    continue
```